egs/fisher_swbd/s5/local/chain/run_tdnn_opgru_1a.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
+# ./local/chain/compare_wer_general.sh tdnn_opgru_1a_sp
+# System                tdnn_opgru_1a_sp
+# WER on eval2000(tg)        12.3
+# WER on eval2000(fg)        12.1
+# Final train prob         -0.097
+# Final valid prob         -0.104
+# Final train prob (xent)        -1.080
+# Final valid prob (xent)       -1.1094
+
+
+set -e
+
+# configs for 'chain'
+stage=12
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=true
+dir=exp/chain/tdnn_opgru_1a # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+decode_dir_affix=
+
+# training options
+leftmost_questions_truncate=-1
+chunk_width=150
+chunk_left_context=40
+chunk_right_context=0
+xent_regularize=0.025
+self_repair_scale=0.00001
+label_delay=5
+# decode options
+extra_left_context=50
+extra_right_context=0
+frames_per_chunk=
+
+remove_egs=false
+common_egs_dir=
+
+affix=
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+# The iVector-extraction and feature-dumping parts are the same as the standard
+# nnet3 setup, and you can skip them by setting "--stage 8" if you have already
+# run those things.
+
+suffix=
+if [ "$speed_perturb" == "true" ]; then
+  suffix=_sp
+fi
+
+dir=$dir${affix:+_$affix}
+if [ $label_delay -gt 0 ]; then dir=${dir}_ld$label_delay; fi
```

Comment:
Please remove this _ld$label_delay decoration.  
In fact, I'm a little confused about where you got this sript because a lot of aspects of it seem to not be the same as other scripts in the fisher_swbd directory.
The lang directory is differently named and you are using a different size tree.
I think something went wrong here.  
Of course, if there is a good reason for these changes, or the results are better, I'll think about that.

Reply:
The experiments are OK under Fisher+Switchboard, all model use same tree "tri6_tree". The misguiding information in the TDNN-OPGRU script is because I used switchboard scripts directly (same for TDNN-PGRU, TDNN-LSTMP). But during training, I just used model training part. This will not influence results.

I will clean up the wrong scripts on Fisher+SWBD and re-upload.